man/loader.conf.xml

```diff
@@ -375,25 +375,27 @@ sbvarsign --attr "${attr}" --key KEK.key --cert KEK.pem --output db.auth db db.e
 
       <varlistentry>
         <term>secure-boot-enroll-action</term>
-
         <listitem>
           <para>Specifies the action to take after the automatic enrollment of secure boot keys is completed.</para>
-
           <variablelist>
             <varlistentry>
-              <term><option>reboot</option></term>
-              <listitem><para>Reboot the system after enrollment. This is the default.</para></listitem>
+              <term>reboot</term>
+              <listitem>
+                <para>Reboot the system after enrollment. This is the default.</para>
+                <xi:include href="version-info.xml" xpointer="v258"/>
```

Comment:
Uh, please drop the version-info for parameters, and update tools/command_ignorelist file to make the missing version info does not fail the test.

Reply:
done!